packages/graph-editor/src/editor/index.tsx

```diff
@@ -539,6 +540,7 @@ export const EditorApp = React.forwardRef<ImperativeEditorRef, EditorProps>(
                     variant={showNodesPanel ? 'primary' : 'invisible'}
                   />
                   {props.menuContent}
+                  <Settings />
```

Comment:
I'm not sure if we should force it being here versus just exposing the setting component so that you can choose to display it or not

Reply:
I now added a prop to control if it should be displayed or not